### ISSUE TITLE
Add DateTime Support

### ISSUE THREAD

**Issue by huythuytq** [signalpoint/date#6]
Hi Tyler.

How to do handle create form date. I create form with type : datetime but error : Cannot read property "widget" of undefine.

**Comment by signalpoint** [comment]
@huythuytq Right now only the Date field is supported, not datetime or datestamp/timestamp. You can either change your field to be just a "Date" field, or you can modify the date.js file to try and accommodate the other types of date fields.

**Comment by huythuytq** [comment]
Yes, my field is "Date". When I create content, it work fine, but I want handle create form, this is my code : 

```
form.elements['appointments_date'] = {
            type : 'date',
            title : 'Appointment Date:',
        };
```

Console log show error : WARNING: _drupalgap_form_render_element_item() - Field date not supported.

**Comment by signalpoint** [comment]
@huythuytq The 'date' type is not a valid form element type. For example, when using a date field on a node, it's actually just a 'select' list(s). Essentially you need to design your element as a select list (or a text field, or other element types: http://drupalgap.org/node/274). You can utilize the 'children' array on your element to attach other widgets to the element to help build the user experience and attach various click handler(s), etc.

**Comment by huythuytq** [comment]
Ok I see, thank you.
